Atomic/syscontainers.py

```diff
@@ -369,6 +368,23 @@ def delete_image(self, image):
         ref = OSTree.parse_refspec(imagebranch)
         repo.set_ref_immediate(ref[1], ref[2], None)
 
+    def inspect_system_image(self, image):
+        repo = self._get_ostree_repo()
+        if not repo:
+            return None
+        imagebranch = SystemContainers._get_ostree_image_branch(image)
+        return self._inspect_system_branch(repo, imagebranch)
+
+    def _inspect_system_branch(self, repo, imagebranch):
+        commit_rev = repo.resolve_rev(imagebranch, False)[1]
+        commit = repo.load_commit(commit_rev)[1]
+
+        tag = ":".join(imagebranch.replace("ociimage/", "").rsplit('-', 1))
+        timestamp = OSTree.commit_get_timestamp(commit)
+
+        return {'Id' : commit_rev, 'RepoTags' : [tag], 'Names' : [], 'Created': timestamp,
```

Comment:
the problem is that the commit having the manifest metadata cannot be mapped to a Docker layer, so I preferred to use the OSTree commit for both layers and the ID, so for now I am using the same ID used for `images list`.

With @yuqi-zhang we are currently discussing about the ID to use in the images list output as well (which currently is the `commit_rev` too), as we are thinking about changing there as well, so that it uses the Docker layer ID, but we didn't yet figure out what ID should be used for the metadata commit, which gives the name to the image.